python/fledge/services/core/api/north.py

```diff
@@ -186,6 +185,7 @@ async def get_north_schedules(request):
         return web.HTTPInternalServerError(reason=msg, body=json.dumps({"message": msg}))
     except Exception as ex:
         msg = str(ex)
+        _logger.error("Get north schedules failed. Found error: {}".format(msg))
         return web.HTTPInternalServerError(reason=msg, body=json.dumps({"message": msg}))
```

Comment:
return or raise?

Reply:
As there are inner try catch is being used. So we need to return early!